paf-mvp-frontend/tests/test-cases/queue.test.ts

```diff
@@ -0,0 +1,88 @@
+import { TestWatcher } from 'jest';
+import { isExportDeclaration } from 'typescript';
+import { Command, CommandQueue, processCommands } from '../../src/utils/queue';
+
+class CommandMocker {
```

Comment:
Use `jest.fn()` instead (more standard)

Reply:
If for some reason you don't want to use `jest.fn()` (but please justify it 😉), consider using a simple attribute rather than a function:
```
func = () => this.called = true;
```
There is no reason that `func` is actually a function, it always returns the same hardcoded value (which happens to be function, but that's unrelated)

Reply:
I don't understand your point 🤨.
I was not discussing whether it should be mocked or not: you _are_ already mocking it, just not using the standard way to do it with Jest.